### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by theRealPadster**

### CODE REVIEW

**Comment by theRealPadster** [comment]
@xerta555 I think this PR is what we want. Could you remove the () around the new strings?

File: packages/marketplace/src/resources/locales/fr.json [hunk 1]
```diff
@@ -44,24 +44,24 @@
     },
     "devTools": {
       "title": "Outils de développeurs de thèmes",
-      "noThemeInstalled": "(Error: No marketplace theme installed)",
-      "noThemeManifest": "(Error: No theme manifest found)",
-      "colorIniEditor": "(Color.ini Editor)",
+      "noThemeInstalled": "(Erreur: Aucun thème du marché n’est installé)",
+      "noThemeManifest": "(Erreur: Aucun manifeste de thème trouvé)",
+      "colorIniEditor": "(Éditeur Color.ini)",
       "colorIniEditorPlaceholder": "([your-color-scheme-name])",
```
Comment (theRealPadster) [inline packages/marketplace/src/resources/locales/fr.json:50]: This one would be good too if you're able

Comment (xerta555) [reply packages/marketplace/src/resources/locales/fr.json]: > This one would be good too if you're able

Done 👍🏻

File: packages/marketplace/src/resources/locales/fr.json [hunk 2]
```diff
@@ -44,24 +44,24 @@
...
     "readmePage": {
-      "loading": "(Loading...)",
-      "errorLoading": "(Error loading README)",
-      "github": "(GitHub)",
-      "remove": "(Remove)",
-      "install": "(Install)"
+      "loading": "Chargement...",
+      "errorLoading": "Erreur lors du chargement du README",
+      "github": "GitHub",
+      "remove": "Supprimer)",
```
Comment (kyrie25) [inline packages/marketplace/src/resources/locales/fr.json:61]:
```suggestion
      "remove": "Supprimer",
```

**Pull request merged**